app/pb_kits/playbook/pb_text_input/_text_input.scss

```diff
@@ -8,51 +8,52 @@
 
   .text_input_wrapper {
     margin-bottom: 1rem;
-    > input:first-child {
-      box-sizing: border-box;
-      border: 1px solid $border-light;
-      border-radius: $border-rad-light;
-      display: block;
-      width: 100%;
-      color: $text_lt_default;
-      background: $white;
-      padding: $space_xs $space_sm;
-      transition: 0.2s all;
-      outline: none;
-      font-size: $font_small;
-      margin: .1rem 0 0;
-
-      &:focus {
-        border-color: $primary-action;
-      }
-
-      &::-webkit-input-placeholder {
-        color: $text_lt_lighter;
-        opacity: 1;
-      }
-
-      &:-ms-input-placeholder {
-        color: $text_lt_lighter;
-        opacity: 1;
-      }
-
-      &::-ms-input-placeholder {
-        color: $text_lt_lighter;
-        opacity: 1;
-      }
-
-      &::placeholder {
-        color: $text_lt_lighter;
-        opacity: 1;
-      }
-
-      &:-ms-input-placeholder {
-        color: $text_lt_lighter;
-      }
-
-      &::-ms-input-placeholder {
-        color: $text_lt_lighter;
-      }
+  }
+
+  .text_input {
+    border: 1px solid $border-light;
+    border-radius: $border-rad-light;
+    display: block;
+    width: 100%;
+    color: $text_lt_default;
+    background: $white;
+    padding: $space_xs $space_sm;
+    transition: 0.2s all;
+    outline: none;
+    font-size: $font_small;
+    margin: .1rem 0 0;
```

Comment:
If you pass a block input it won't have this class. You will likely have a similar pattern as Terry made